docs/source/introduction/quickstart.ipynb

```diff
@@ -22,45 +22,56 @@
     "\n",
```

Comment:
<p>Is there a reason you deleted the remark about preconditioning? This makes us quite open to the criticism that we are "reinventing the wheel", since this example is really just preconditioning.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>It's a quickstart for code, so some comments are more distracting at that point, which is why I took it out originally. I added back a short version of the original remark, to keep the flow. "The prior on the inverse plays a similar role to the preconditioner for classic linear solvers." </p>